File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int length=0;
        int n=s.length()-1;
        while(!isalnum(s[n])){
            n--;
        }

        for(int i=n;i>=0;i--){
            if(s[i]==' '){
                break;
            }else{
                length++;
            }
        }
        return length;  
    }
};

File: 58-length-of-last-word/length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    arr=s.split(" ");
    n=arr.length;
    i=n-1;
    while(arr[i]===""){
        i--;
    }
    word=arr[i]
    return word.length;
};